spec/requests/api/v1/events/events_spec.rb

```diff
@@ -80,3 +81,44 @@
     expect(response.code.to_i).to eq 422
   end
 end
+
+describe 'PATCH /v1/events/:id' do
+  it 'updates the event attributes' do
+    event = create(:event, name: 'Old name')
+    new_name = 'New name'
+    event_params= {
+      name: new_name,
+      owner: {
+        device_token: event.owner.device_token
+        },
```

Comment:
This and the next `}` should align with the line that opened the block / section